### ISSUE TITLE
imgmount2 & imgmount3 from url?

### ISSUE THREAD

**Issue by imneckro** [nbarkhina/DosWasmX#7]
Hello. 
Is it possible to imgmount 2 and imgmount 3 from url?

reading below i understand you can mount only default img.

var DOSWASMSETTINGS = {
    CLOUDSAVEURL: "",
    ISOURL: "",
    DEFAULTIMG: "https://www.yourwebsite.com/hdd.img"
}

Many thanks in advance.

**Comment by nbarkhina** [comment]
Are you trying to mount multiple drives, is that the reason? Currently no it only supports a single drive. What is it you are trying to do, is there a reason you need multiple drives? Current max drive size is 2gb so should be sufficient for most scenarios.

**Comment by imneckro** [comment]
Is it possible to mount an hdd.img as an iso with win95 c.img as an hdd?

**Comment by nbarkhina** [comment]
Try renaming to iso as "hdd.iso" instead of "hdd.img". Then boot into windows and from the Advanced menu click on "Load CD" and select the "hdd.iso" file

**Comment by imneckro** [comment]
I tried Load cd and boot in windows,But there are no cdrom inside win95..

**Comment by nbarkhina** [comment]
If the iso mounts successfully windows should detect it. Maybe try loading the windows 95 ISO because maybe it's missing some drivers? Otherwise you may need to reinstall windows 95 using the emulator.

**Comment by imneckro** [comment]
Thanks again for all your help. 
I reinstalled win95 and last I have managed to read iso as cdrom inside win95.
By the way changing dosbox-x-for-web.conf does it have any impact at all or this conf being skipped?

I am asking this question because no matter what settings I put in this conf, win95 emulation is very slow.

**Comment by nbarkhina** [comment]
It does use the dosbox-x-for-web.conf however some of the values get replaced during load in the script.js file. The performance is mainly affected by the cpu setting - you can play around with options like auto, max or a fixed value. but generally the performance will be affected by the speed of your machine and particular software you are trying to run.

**Comment by imneckro** [comment]
thanks once again for your help.
Im attaching my [c.img](https://t.ly/Ut92i) to try it  out.
You'll see slow fps permormance ingame.

**Comment by nbarkhina** [comment]
yeah it runs slow for me too. my guess is you would need a pretty powerful computer to run this in the browser. I'm frankly amazed it runs at all :)

**Comment by imneckro** [comment]
in windows with DOSBox-0.74-3 runs just fine though
